client/components/findStar.jsx

```diff
@@ -59,6 +60,30 @@ class FindStar extends Component {
         event.preventDefault();
     }
 
+    handleRemove(){
+        var data = {username: 'test1', favorite: this.state.click};
+        fetch('http://localhost:5000/removeFav', {
+            credentials: 'same-origin',
+            method: 'POST', // 'GET', 'PUT', 'DELETE', etc.
+            body: JSON.stringify(data), // Coordinate the body type with 'Content-Type'
+            headers: new Headers({
+                'Content-Type' : 'application/json',
+                'Accept': 'application/json'
+            })
+        }).then((result) => {
+            if (result.status === 200) {
+                console.log('removed from the favorite list');
+            } else if (result.status === 401) {
+                console.log('Selected celebrity is already removed from the list please add first');
```

Comment:
`elected celebrity is already removed from the list. Please add first`
Please make sure when you see the error message, it is grammatically correct